src/js/letters/reducers/index.js

```diff
@@ -23,16 +27,34 @@ function letters(state = initialState, action) {
       // the various error scenarios
       return set('lettersAvailable', false, state);
     case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS':
+      // Create object for which options to include in post for summary letter
```

Comment:
Since `action.data.data.attributes.benefitInformation` is so horribly ugly and used 2x below, we may want to define `const benefitInfo = ...` here.